lnrpc/rpc.proto

```diff
@@ -1930,13 +1930,15 @@ message ChannelEventUpdate {
         ChannelCloseSummary closed_channel = 2 [ json_name = "closed_channel" ];
         ChannelPoint active_channel = 3 [ json_name = "active_channel" ];
         ChannelPoint inactive_channel = 4 [ json_name = "inactive_channel" ];
+        PendingUpdate pending_open_channel = 6 [json_name = "pending_open_channel"];
```

Comment:
Last thing, sorry that I missed this! This should be 5 rather than 6?
You also need to run `make rpc` after this most recent change, some things change when I run it.

Reply:
> Last thing, sorry that I missed this! This should be 5 rather than 6?

This was deliberate from my side -- maybe misunderstanding how protobufs works, but `UpdateType type` is number 5.

https://github.com/lightningnetwork/lnd/blob/ab04a8b3c0566a9321f5f672c81c6db719efbb15/lnrpc/rpc.proto#L1944 

>  You also need to run make rpc after this most recent change, some things change when I run it.

Weird, I will do that.

Reply:
@carlaKC You're right, changes to `rpc.swagger.json` is missing.
Quite a lot of other changes seem to happen as well to the file, so I suspect I have the wrong version of some tool. I'll work on fixing that.

Reply:
Okay `rpc.swagger.json` has been updated.

Reply:
> This was deliberate from my side -- maybe misunderstanding how protobufs works, but UpdateType type is number 5.

Yes that's correct, missed that field when I was scanning it.